config.mk

```diff
@@ -1,6 +1,16 @@
+ifeq ($(CLOUD),azure)
+
+SUBSCRIPTION_ID := $(shell az account list | jq -rj '.[0].id')
+REGION := eastus
+RESOURCE_GROUP := hail-dev
+CONTAINER_REGISTRY_NAME := $(RESOURCE_GROUP)
+SHARED_GALLERY_NAME := batch
+DOCKER_PREFIX := $(CONTAINER_REGISTRY_NAME).azurecr.io
+
+else
+
```

Comment:
Ya I would like if possible for this PR not to touch `config.mk`. I think it's alright for now if some of these values were hard coded into the shell script, one of my following PRs will include harmonizing all of this.